src/index.html

```diff
@@ -21,7 +21,425 @@
     />
   </head>
 
-  <body>
-    <h1>Catalog</h1>
+  <body class="body">
+    <header class="header">
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          class="logo__image"
+          alt="Moyo Logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li>
+            <a
+              href="#"
+              class="nav__item is-active"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li>
+            <a
+              href="#"
+              class="nav__item"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li>
+            <a
+              href="#"
+              class="nav__item"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li>
+            <a
+              href="#"
+              data-qa="nav-hover"
+              class="nav__item"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li>
+            <a
+              href="#"
+              class="nav__item"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li>
+            <a
+              href="#"
+              class="nav__item"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li>
+            <a
+              href="#"
+              class="nav__item"
+            >
+              Photo
+            </a>
+          </li>
+
+          <li>
+            <a
+              href="#"
+              class="nav__item"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="container">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <div class="card__logo"></div>
+        <div class="card__description">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </div>
+        <div class="card__code">Product code: 195434</div>
+        <div class="card__stars">
+          <div class="stars">
+            <div class="stars__container">
+              <div class="stars stars--4">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+            </div>
+
+            <div class="stars__reviews">Reviews: 5</div>
+          </div>
+        </div>
+
+        <div class="card__price">
+          <div class="card__price-name">Price:</div>
+          <div class="card__price-amount">$2,199</div>
+        </div>
+        <div
+          class="card__button"
+          data-qa="card-hover"
```

Comment:
The `data-qa` attribute is used for testing purposes and should not be present in the production code unless it's being used for automated tests.